my-app/src/reducers/todoListReducer.js

```diff
@@ -0,0 +1,71 @@
+export const initialState = {
+  todos:[
+    {
+      item: 'Learn about reducers',
+      completed: false,
+      id: 3892987589
+    },
+    {
+      item: 'Do reducers project',
+      completed: false,
+      id: 3892987590
+    },
+    {
+      item: 'Celebrate',
+      completed: false,
+      id: 3892987591
+    }
+  ]
+}
+
+export const todoListReducer = (state, action) => {
+  switch (action.type) {
+    case 'ADD_ITEM':
+      const newItem = {
+        item: action.payload,
+        completed: false,
+        id: new Date()
+      }
+      return {
+        ...state,
+        todos: [...state.todos, newItem]
+      };
+    case 'COMPLETE_ITEM':
+      const completedItems = state.todos.map(item => {
+        if(item.id === action.payload) {
+          return {
+            ...item,
+            completed: !item.completed
+          }
+        } else {
+          return item;
+        }
+      })
+      completedItems.map(item => {
+        if(item.id === action.payload) {
+          document.getElementById(item.id).classList.toggle('todo-list-item-true');
+          console.log(action.payload);
+          document.getElementById(item.id*2).classList.toggle('button-true');
+        }
+      })
+      return {
+        ...state,
+        todos: completedItems
+      };
+    case 'CLEAR_ITEMS':
+      console.log(document.querySelectorAll('.todo-list-item-true'));
```

Comment:
This could be much simpler with a filter:
```suggestion
      const newArray = state.todos.filter(todo => todo.completed === false)
      return {
      ...state, 
      todos: newArray
      }
```